### ISSUE TITLE
Is it safe to use one of the tags for packaging?

### ISSUE THREAD

**Issue by hwittenborn** [minecraft-linux/msa-manifest#3]
I'd like to start packaging this minecraft launcher for Debian, and I'm looking into getting everything compiled correctly. I saw that the last commit was made in March of 2021, but the latest tag is only from December of 2018. Is there a reason the latest tag is so old in comparison to the latest commit?

**Comment by ChristopherHX** [comment]
I created a tag for the current used version should be save to use now.
However I still need to investigate why xbox sign in broke in 2021. This component is unused for Minecraft 1.16.20+ and is not needed for xbox sign in anymore.

I stopped packaging deb in 2020 due to lack of an apt repository.

**Comment by hwittenborn** [comment]
> This component is unused for Minecraft 1.16.20+ and is not needed for xbox sign in anymore.

So I don't need to package this if I want to package the launcher?

**Comment by ChristopherHX** [comment]
Yes you don't need package it, It's optional. Only required for xbox live <= Minecraft 1.16.10, you can select that old versions from the versions picker of the launcher and they open if they are >= 1.13 in v0.3.4
